api/src/main/kotlin/edu/wgu/osmt/collection/CollectionEsRepo.kt

```diff
@@ -77,21 +85,25 @@ class CustomCollectionQueriesImpl @Autowired constructor(
         }
     }
 
+    /**
+     * TODO upgrade to ElasticSearch v8.7.x api style; see KeywordEsRepo.kt & FindsAllByPublishStatus.kt
+     */
     override fun byApiSearch(
         apiSearch: ApiSearch,
         publishStatus: Set<PublishStatus>,
         pageable: Pageable
     ): SearchHits<CollectionDoc> {
-        val nsq: NativeSearchQueryBuilder = NativeSearchQueryBuilder().withPageable(Pageable.unpaged())
+        val nsqb1 = NativeSearchQueryBuilder().withPageable(Pageable.unpaged())
```

Comment:
Is there meaning in the ordinal "1" on the variable name? It implies that there will be more?